engine/topology/src/main/java/org/opennms/oce/engine/topology/ActionManager.java

```diff
@@ -48,10 +53,24 @@ public ActionManager(TopologyEngine topologyEngine) {
     public void createIncidentOnFailure(Group group) {
         LOG.info("Got failure for: {}", group);
 
+        ModelObject owner = group.getOwner();
         IncidentBean incident = new IncidentBean(UUID.randomUUID().toString());
+        incident.setModelObject(owner);
         group.getMembers().stream()
                 .flatMap(mo -> mo.getAlarms().stream())
                 .forEach(incident::addAlarm);
         topologyEngine.getIncidentHandler().onIncident(incident);
+        // Impact the Group
+        synthesizeAlarm(owner);
```

Comment:
Should we always be impacting the owner, or should we maybe make this behavior optional and add another action, leaving it up to the rule to decide?

Reply:
my thinking was 'yes' - at least until we ran into an actual case where it was 'no', but I'm find to pull it into the rules now.
The one concern I have is that it's then extra boilerplate for each rule, and a rule that forgets to include it will become a problem.